python/tskit/trees.py

```diff
@@ -3334,6 +3334,33 @@ def dump_text(
                 )
                 print(row, file=provenances)
 
+    def __repr__(self):
+        ts_rows = [
+            ["Trees", str(self.num_trees)],
+            ["Seq Length", str(self.sequence_length)],
```

Comment:
We can probably say "Sequence Length", I think there's plenty hspace.

Reply:
Done